libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.m

```diff
@@ -67,7 +68,12 @@ - (instancetype)initWithBackgroundSession {
     return self;
 }
 
-- (NSURLSessionDataTask *)sendRequest:(NSURLRequest *)urlRequest dataResponseBlock:(SFDataResponseBlock)dataResponseBlock {
+- (NSURLSessionDataTask *)sendRequest:(NSMutableURLRequest *)urlRequest dataResponseBlock:(SFDataResponseBlock)dataResponseBlock {
+    // Sets Mobile SDK user agent if it hasn't been set already elsewhere.
+    if (![urlRequest.allHTTPHeaderFields.allKeys containsObject:@"User-Agent"]) {
```

Comment:
Setting user agent in SFNetwork.

Reply:
That way it applies to oauth requests as well.

Reply:
Would this be set beforehand for requests originating from `SmartSync`? Where would the `qualifier` come in for `SmartSync` requests?